### PR TITLE
Updated date and time fields

### LINKED ISSUE #5: Input field should be a Date Picker and Time Picker
image

- [ ] Label should read:

> When would you be available for a visit
instead.
- [ ] The input field should be a date picker and time picker.

### PR DESCRIPTION
**Pull request by jveer634**
Closes #5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -122,7 +122,7 @@
                    <div class="field">
                         <label class="label">D.O.B</label>
                     <div class="control">
-                        <input type="text" class="input" placeholder="Your DOB">
+                        <input type="date" class="input" placeholder="Your DOB">
```
Comment (stbensonimoh) [inline index.html:125]: Perfect! :thumbsup:

File: index.html [hunk 2]
```diff
@@ -132,7 +132,7 @@
                      <div class="field">
                         <label class="label">Time available for visit</label>
                         <div class="control">
-                            <input class="input" type="text" name="visit" id="visit" placeholder="When are you available at home?">
+                            <input class="input" type="time" name="visit" id="visit" placeholder="When are you available at home?">
```
Comment (stbensonimoh) [inline index.html]: Please can you use the `<input type="datetime-local">` instead?

**Review by stbensonimoh — APPROVED** [review]
@jveer634 Thanks for the PR. Kindly make the changes and merge.

**Comment by stbensonimoh** [comment]
@jveer634 Don't bother about the netlify checks. I think it's from the deployment settings on Netlify by @anyanime. He will rectify that. It is deploying successfully.

**Comment by jveer634** [comment]
Please have a look.

**Comment by stbensonimoh** [comment]
:+1: Looks great to me!

**Pull request merged**